Solutions/PaloAltoCDL/Hunting Queries/PaloAltoCDLIncompleteApplicationProtocol.yaml

```diff
@@ -0,0 +1,28 @@
+id: 7cbd46ce-5121-11ec-bf63-0242ac130002
+name: PaloAlto - Incomplete application protocol
+description: |
+  'Query shows incomplete application protocol'

```

Comment:
Can we please check the detection logic and the description. From the logic seems like we are not checking anything related to incomplete application protocol. Please double check.